test/sync.js

```diff
@@ -186,6 +186,13 @@ tape('sync: trying to sync to unknown peer', function (t) {
   })
 })
 
+tape('sync: leave even if swarm is undefined', function (t) {
+  var api1 = helpers.createApi(null)
+  t.notOk(api1.sync.leave())
```

Comment:
`t.notOk()` is redundant, since `leave()` returns `undefined` no matter its input.